testing/basic_agent_tests.py

```diff
@@ -0,0 +1,20 @@
+"""
+basic_tests.py - Using testinfra to run tests on ansible playbook rke2-ansible
+"""
+
+import testinfra
+
+
+def test_rke2_config(host):
+    rke2_config = host.file("/etc/rancher/rke2/config.yaml")
+    assert rke2_config.contains("token:")
+    assert rke2_config.contains("server:")
+    assert rke2_config.user == "root"
+    assert rke2_config.group == "root"
+    assert rke2_config.mode == 0o640
```

Comment:
This looks weird, are you sure you didn't mean 0644?

Reply:
That's the way the python reads the octal.  This is technically correct and works as intended.

Reply:
Doh, yeah my bad.